### PR TITLE
Add VAT fields in transactions

### PR DESCRIPTION
**Pull request by tpayet**

### CODE REVIEW

File: lib/qonto/model/transaction.rb [hunk 1]
```diff
@@ -1,20 +1,10 @@
...
-      attr_accessor :operation_type
-      attr_accessor :currency
-      attr_accessor :local_currency
-      attr_accessor :settled_at
-      attr_accessor :label
-      attr_accessor :status
-      attr_accessor :transaction_id
-      attr_accessor :emitted_at
-      attr_accessor :note
+      attr_accessor :amount, :amount_cents, :local_amount, :local_amount_cents,
```
Comment (scharrier) [inline lib/qonto/model/transaction.rb]: 👍 for calling `attr_accessor` only once, but I think that I prefer to keep one attribute / line, as it seems to be easier to read. WDYT?

Btw, don't you think we should alpha sort attributes?

**Review by scharrier — COMMENTED** [review]
Thanks for your contribution :)

LGTM beside a small comment. As this part isn't tested at all, would you mind adding specs to test that returned models has the good attributes? (If this is a problem I'll add these tests)

File: lib/qonto/model/transaction.rb (hunk 1 above)
Comment (tpayet) [reply lib/qonto/model/transaction.rb]: I corrected it with one `attr_accessor` per line and alpha sorted them

**Comment by tpayet** [comment]
Pleas tell me if the spec I wrote is okay for you

File: spec/qonto/client_spec.rb [hunk 2]
```diff
@@ -102,6 +103,23 @@
       response = client.list_transactions(bank_account: account)
 
       expect(response.first).to be_a(Qonto::Model::Transaction)
+      expect(response.first.instance_variables).to eq(%i[@amount
```
Comment (scharrier) [inline spec/qonto/client_spec.rb]: I think that what we really want to test here, is that each attribute present in the JSON response is accessible through the model interface, and that it has been set at the good value.

What do you think of instead adding a test per attribute we want to test ? This is a but longer to write, but may be safer on a long term point of view (= we are sure that everything is present and correctly set). And maybe easier to read.

Example:
```
expect(response.first.amount).to eq(0.42)
expect(response.first.amount_cents).to eq(42)
# ...
```

Do you agree ?

File: spec/qonto/client_spec.rb [hunk 3]
```diff
@@ -1,3 +1,4 @@
+require 'byebug'
```
Comment (scharrier) [inline spec/qonto/client_spec.rb]: Why do we add that line ?

**Review by scharrier — CHANGES_REQUESTED** [review]
If you agree with me, we could do the 2 changes I proposed and I'll merge and release a new version. Thanks again :)

**Comment by scharrier** [comment]
Thanks again for your help @tpayet, I just released a new version of the gem (0.3.0).

**Pull request merged**